2.7.0/todo/result_onnx.md

```diff
@@ -26,15 +25,23 @@ The categories below are as follows:
 * not user facing: All commits that are not public end-user facing and hence should be dropped from the release notes
 
 ## onnx
+
 ### bc breaking
+
 - [ONNX] Set USE_EXPERIMENTAL_LOGIC to True ([#137296](https://github.com/pytorch/pytorch/pull/137296))
+
 ### deprecation
+
 - [ONNX] Create deprecation warning on dynamo_export ([#146003](https://github.com/pytorch/pytorch/pull/146003))
 - [ONNX] Create deprecation warning on dynamo_export ([#146425](https://github.com/pytorch/pytorch/pull/146425))
```

Comment:
1. These look the same

2. please add similar deprecation descriptions based on the same link I linked above!